app/models/comment.rb

```diff
@@ -1,2 +1,10 @@
 class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  attribute :text, :text
+  after_create :update_post_comments_counter
+  def update_post_comments_counter
+    post.increment!(:comments_counter)
```

Comment:
- [ ] OPTIONAL
Following best practices `update_comments_counter` should be marked as _private_ after implementing the `after_create` method.